### PR TITLE
Doc 302 document that 1st run w datastore is

### PR DESCRIPTION
**Pull request by oojo12**
# What does this PR do?
Add important note about performance related to accessing datastores

### CODE REVIEW

**Comment by oojo12** [comment]
@rlizzo can you give this a quick review?

File: docs/features/runs/attaching-datastores.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ You can mount a datastore to a run to make your experiments run faster! By defau
 grid run main.py --my_data_path /datastores/cats/1
 ```
 
+:::note
+  When using datastores the data are requested from S3 and loaded on demand. These data results are then cached to local disk. Because of this you will see slower data
+  retrieval times during the first epoch of a run.
+:::note
```
Comment (essiequoi) [inline docs/features/runs/attaching-datastores.md:20]: I would rephrase this as "Because of this you will see faster retrieval times after the first epoch of a run"

**Review by essiequoi — CHANGES_REQUESTED** [review]

File: docs/features/runs/attaching-datastores.md (hunk 1 above)
Comment (oojo12) [reply docs/features/runs/attaching-datastores.md:19]: done.

**Review by essiequoi — APPROVED** [review]

**Comment by essiequoi** [comment]
just need tech review then good to go for me

File: docs/features/runs/attaching-datastores.md (hunk 1 above)
Comment (rlizzo) [reply docs/features/runs/attaching-datastores.md:19]: This is not true at all. The files can be temporarily cached in memory, but we don't cache files on disk.

**Review by rlizzo — CHANGES_REQUESTED** [review]

File: docs/features/runs/attaching-datastores.md (hunk 1 above)
Comment (essiequoi) [reply docs/features/runs/attaching-datastores.md:19]: @oojo12 fyi

**Comment by oojo12** [comment]
Closing in favor of #179
